### ISSUE TITLE
How to change the config to use multi appsetting.json

### ISSUE THREAD

**Issue by benzhipan** [AzureAD/microsoft-identity-web#2404]
### Microsoft.Identity.Web Library

Microsoft.Identity.Web.TokenAcquisition

### Microsoft.Identity.Web version

2.13.3

### Web app

Not Applicable

### Web API

Protected web APIs (validating tokens)

### Token cache serialization

Not Applicable

### Description

Microsoft.Identity.Web.TokenAcquirerFactory

Description of the issue
[The project](https://github.com/AzureAD/microsoft-identity-web/blob/master/tests/DevApps/daemon-app/daemon-console-calling-msgraph/Program.cs) works fine, but I need to have more environments to run this.
How to change Configuration to use appsettings.Development.json, Configuration is protected set

### Reproduction steps

https://github.com/AzureAD/microsoft-identity-web/blob/master/tests/DevApps/daemon-app/daemon-console-calling-msgraph/Program.cs
appsettings.Development.json

### Error message

How to change Configuration to use appsettings.Development.json, Configuration is protected set

### Relevant code snippets

```csharp
https://github.com/AzureAD/microsoft-identity-web/blob/master/tests/DevApps/daemon-app/daemon-console-calling-msgraph/Program.cs
```

### Expected behavior

change config to use appsettings.Development.json or appsettings.Production.json

**Comment by jmprieur** [comment]
would you like to pickup all the `appsettings.*.json` files, @benzhipan ?

**Comment by benzhipan** [comment]
Yes, That would be nice or change Configuration propery of "tokenAcquirerFactory"  to Public, so I can set it.
`var tokenAcquirerFactory = TokenAcquirerFactory.GetDefaultInstance();
tokenAcquirerFactory.Services.AddMicrosoftGraph();
var sp = tokenAcquirerFactory.Build();
`

**Comment by jacktise** [comment]
This would be a massive help, we're running multiple environments that require different token acquisition configurations and it's a massive headache having to make sure we change `appsettings.json` for each deployment.

**Comment by KAGUser** [comment]
Any resolution to this?  I have the same issue where TokenAcquirerFactory.GetDefaultInstance() ignores configuration values in environment-specific appsettings.json.  I posted the question in the link below.  As a temporary workaround, before calling GetDefaultInstance, I call a routine to replace appsetting.json values with the specific environment values.

https://stackoverflow.com/questions/78030179/tokenacquirerfactory-getdefaultinstance-ignores-configuration-values-in-enviro
